smartcontracts/sandbox/contracts/interfaces/IRouter.sol

```diff
@@ -33,21 +24,19 @@ interface IRouter {
     /// @return address of fleet manager
     function fleetManager() external view returns (address);
 
-    /// @notice register project data receiver
-    /// @param _projectId project id
-    /// @param _receiver project data reveiver
-    function register(uint256 _projectId, address _receiver) external;
-
-    /// @notice unregister project data receiver
-    /// @param _projectId project id
-    /// @param _receiver project data reveiver
-    function unregister(uint256 _projectId, address _receiver) external;
-
     /// @notice submit data to project
     /// @param _projectId project id
     /// @param _receiver project data reveiver
-    /// @param _data data
-    function submit(uint256 _projectId, address _receiver, bytes calldata _data) external;
+    /// @param _batchMR batch merkle root
+    /// @param _devicesMR devices merkle root
+    /// @param _zkProof zk proof
+    function submit(
+        uint256 _projectId,
+        address _receiver,
+        bytes32 _batchMR,
```

Comment:
_batchMR, devicesMR and _zkProof should packed into on `bytes _data` parameter, this can make Router more flexibility.